core-ui/src/components/Extensibility/components/ResourceSchema.js

```diff
@@ -103,14 +103,27 @@ export const ResourceSchema = ({ resource, setResource, schema, path }) => {
 
   if (isEmpty(schema)) return null;
 
-  const schemaMap = createOrderedMap(schema);
+  let newschema = schema;
+  delete newschema.properties.metadata;
+
+  newschema = {
+    ...newschema,
+    properties: { metadata: METADATA_SCHEMA, ...newschema.properties },
```

Comment:
instead of deleting metadata four lines higher, maybe just go with a reverse order
```suggestion
    properties: { ...newschema.properties, metadata: METADATA_SCHEMA },
```
???

Reply:
The order is important. That's why I remove it above